lib/zinnia.js

```diff
@@ -187,14 +187,20 @@ const catchChildProcessExit = async ({
       // Store the full error message including stdout & stder in the top-level `details` property
       Object.assign(moduleErr, { details: err.message })
 
-      // Apply a custom rule to force Sentry to group all issues with the same module & exit code
-      // See https://docs.sentry.io/platforms/node/usage/sdk-fingerprinting/#basic-example
-      Sentry.withScope(scope => {
-        scope.setFingerprint([message])
-        maybeReportErrorToSentry(moduleErr)
-      })
+      if (err.signalCode && ['SIGTERM', 'SIGKILL', 'SIGINT'].includes(err.signalCode)) {
+        // These signal codes are triggered when somebody terminates the process from outside.
+        // It's not a problem in Zinnia, there is nothing we can do about this.
+        // Don't report this error to Sentry.
+        Object.assign(err, { reportToSentry: false })
+      } else {
+        // Apply a custom rule to force Sentry to group all issues with the same module & exit code
+        // See https://docs.sentry.io/platforms/node/usage/sdk-fingerprinting/#basic-example
+        Sentry.withScope(scope => {
+          scope.setFingerprint([message])
+          maybeReportErrorToSentry(moduleErr)
+        })
+      }
     }
-    throw err
```

Comment:
this line looks like an unintentional logic change, or is it not?

Reply:
Ouch, it was unintentional! I was wondering why are we setting `{ reportToSentry: false }` when the error is not thrown later.

Thanks for catching this 🙏🏻

Reply:
This is still a logic change, see diff: Now we don't throw on AbortError

Reply:
🙈 

I should slow down and review my diffs before asking for code review. 🤦🏻